eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/processors/SubSnpNoHgvsToClusteredVariantProcessor.java

```diff
@@ -72,4 +80,48 @@ public DbsnpClusteredVariantEntity process(SubSnpNoHgvs subSnpNoHgvs) throws Exc
         return variantEntity;
     }
 
+    /**
+     * If the RefSnp type cannot be determined because none of the variants are type INS, DEL or INDEL, the type INDEL
+     * is assigned by default and the SubSnps would be declustered in
+     * {@link SubSnpNoHgvsToDbsnpVariantsWrapperProcessor }
```

Comment:
Suggested rewording: 

> If the RefSnp type cannot be determined because none of the associated SubSnps are type INS, DEL or INDEL, the type INDEL is assigned by default, and the SubSnps should be declustered by {@link SubSnpNoHgvsToDbsnpVariantsWrapperProcessor}.